### PR TITLE
Sravan branch

### PR DESCRIPTION
**Pull request by SravanKumarL**

### CODE REVIEW

File: src/client/components/UI/DataGrid/DataGridContainer.js [hunk 1]
```diff
@@ -137,30 +137,13 @@ export default class DataGridContainer extends React.PureComponent {
...
-                    </Button>
-                        <Button
-                            style={buttonStyle}
-                            color='default'
-                            variant='raised'
-                            disabled={disabled}
-                            onClick={this.printClickedHandler}>
-                            <PrintIcon style={{ margin: '0px 10px', fontWeight: 'bold' }} /> Print
-                        </Button>
+                        <FilterButton disabled={disabled} onFilterClicked={this.filterClickedHandler} displayFilter={displayFilter} />
```
Comment (alurakhil24) [inline src/client/components/UI/DataGrid/DataGridContainer.js]: Very good bro :)

File: src/client/components/UI/DataGrid/DataGridContainer.js [hunk 2]
```diff
@@ -178,4 +161,31 @@ export default class DataGridContainer extends React.PureComponent {
...
+    <Button
+        style={getButtonStyle(disabled)}
+        color='default'
+        variant='raised'
+        disabled={disabled}
+        onClick={onPrintClicked}>
+        <PrintIcon style={{ margin: '0px 10px', fontWeight: 'bold' }} /> Print
+    </Button>
+);
+const getButtonStyle = disabled => {
```
Comment (alurakhil24) [inline src/client/components/UI/DataGrid/DataGridContainer.js]: Very neat and clean

File: src/client/components/UI/DataGrid/DataGridContainer.js (hunk 1 above)
Comment (SravanKumarL) [reply src/client/components/UI/DataGrid/DataGridContainer.js]: Thank you :)

File: src/client/components/UI/DataGrid/DataGridContainer.js (hunk 2 above)
Comment (SravanKumarL) [reply src/client/components/UI/DataGrid/DataGridContainer.js]: Thank you :-)

**Pull request merged**